includes/import-products.php

```diff
@@ -18,11 +18,26 @@
  * @throws Exception
  */
 function dw_import_products() {
+	global $dw_options;
+
 	$client  = RestApiClientFactory::getClient();
 	$courses = $client->getCourseInstances(new DateTime(), null) ?? [];
-
 	$createdProducts = [];
 
+	if( $dw_options['ccvCode'] && $dw_options['ccvCode'] !== "") {
+		$codesToFilter = explode(';', $dw_options['ccvCode']);
+
+		$courses = array_filter($courses, function($course) use ($codesToFilter) {
+			foreach($codesToFilter as $code) {
+				if($code !== '' && $course['ccvCode'] === $code) {
```

Comment:
Do we need to make this case insensitive?

Reply:
Maybe also trim the codes, if users do `CCV1; CCV2 ; CCV3`?

Reply:
trimmed and lowercased the lot